span/bode.py

```diff
@@ -20,18 +20,18 @@ def __post_init__(self):
         self.timeArray = np.linspace(1/self.samplerate, self.voltageOut.size/self.samplerate, self.voltageOut.size)
     
     @staticmethod
-    def integral(x, y):
+    def integral(x: np.ndarray, y: np.ndarray) -> float:
         return trapezoid(y, x)
 
     @staticmethod
-    def FFT(voltage):
+    def FFT(voltage: np.ndarray) -> np.ndarray:
         return np.fft.fftshift(np.fft.fft(voltage))
     
     @staticmethod
-    def freqs(voltage, samplerate):
+    def freqs(voltage: np.ndarray, samplerate: float) -> np.ndarray:
```

Comment:
Whilst `samplerate` can technically be a `float` in this method, it is expected to be `int` every other time it occurs in this class